### PR TITLE
Replace gas price with fee

### PR DESCRIPTION
**Pull request by gterzian**
Part of https://github.com/massalabs/massa/pull/3173#discussion_r1014817564

### CODE REVIEW

File: assembly/std/index.ts [hunk 1]
```diff
@@ -258,7 +258,7 @@ export function sendMessage(
     validityStartThread,
     validityEndPeriod,
     validityEndThread,
-    maxGas,
+ rawFees,
```
Comment (Thykof) [inline assembly/std/index.ts:261]: You can use `npm run fmt` to format the code and see linting issues about jsdoc in order to make the CI green.

**Review by Thykof — CHANGES_REQUESTED** [review]

**Review by Thykof — APPROVED** [review]

**Comment by gterzian** [comment]
Should we merge this? What is the procedure for that? @Thykof

**Comment by Thykof** [comment]
@gterzian I am asking @gregLibert for review

**Review by gregLibert — APPROVED** [review]

**Comment by gregLibert** [comment]
thanks @gterzian !

**Comment by qdrn** [comment]
This change will only be effective in the testnet 17. This makes the latest version of massa-as-sdk not compatible with the current testnet (used for hackathon). Is that a problem ? @gregLibert @Thykof

**Comment by gregLibert** [comment]
Yes it is. We need to reverse...

**Comment by gterzian** [comment]
Ok, restored the branch and will re-open later when we can merge...

**Pull request merged**